### ISSUE TITLE
Only writing "template" in the output file when using a template.

### ISSUE THREAD

**Issue by ttpvsofficial** [radusuciu/traktor_nowplaying#9]
When starting traktor_nowplaying via batch file with the options: "-o nowplaying.txt --template template" ,
it will only write the string "template" into the output file.
Note that the function trigger is still working (Traktor Play Count timer) 
![traktornowplaying-github-issue-post-screenshots](https://user-images.githubusercontent.com/101729421/158622638-5e34412f-07cd-4675-adc6-fe60c6b6b2d0.PNG)

**Comment by succinct** [comment]
It looks like you actually want to use `--template <path_to_template>` rather than `--format template`. Currently you're telling it to use the string you pass in: "template". It's doing exactly that, which isn't super useful :)

**Comment by ttpvsofficial** [comment]
@succinct Thanks in advance!

U are right, my usage was wrong...

Actually the argument "template.txt" was not an issue.
I used the wrong option --format instead of --template.

Also I had to discard the HTML formatting inside the template file because it somehow pastet the Structure-Stings like "< BODY >" etc.

So for people with that kind of usage issues I ended up with:

![traktornowplaying-github-issue-post-screenshots-solution](https://user-images.githubusercontent.com/101729421/159116474-e3935c9c-5fc6-46cf-a1a6-57d7860aaaeb.PNG)


That might not be a good solution but for me it worked out due to only wanting to add spaces between the "Artist" and "Title" variables. Further formatting is done directly in OBS Studio.
